### PR TITLE
[KARAF-6501] Restoring the wiring of fragment bundles with multiple hosts

### PR DESCRIPTION
**Pull request by nantunes**
Details in https://jira.apache.org/jira/browse/KARAF-6501.

### CODE REVIEW

File: features/extension/src/main/java/org/apache/karaf/features/extension/StoredWiringResolver.java [hunk 1]
```diff
@@ -111,7 +134,7 @@ synchronized void update(Bundle bundle) {
 
     synchronized void delete(Bundle bundle) {
         if (wiring.get(bundle.getBundleId()) != null) {
-            wiring.get(bundle.getBundleId()).delete(path);
+            wiring.remove(bundle.getBundleId()).delete(path);
```
Comment (nantunes) [inline features/extension/src/main/java/org/apache/karaf/features/extension/StoredWiringResolver.java]: This change is unrelated with the bug. I noticed it when writing the unit tests. I wasn't expecting the deleted bundle to remain in the in memory wiring map. If it was intentional I can remove it from the commit.

**Comment by nantunes** [comment]
I'm not sure what other files have an unapproved license... 🤔

**Comment by jbonofre** [comment]
retest this please

File: features/extension/src/test/resources/wirings/5 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+osgi.ee; (&(osgi.ee=JavaSE)(version=1.8))
```
Comment (jbonofre) [inline features/extension/src/test/resources/wirings/5]: This file doesn't contain the license header. You should exclude from rat configuration.

File: features/extension/src/test/resources/wirings/9 [hunk 3]
```diff
@@ -0,0 +1,2 @@
+osgi.wiring.package; (&(osgi.wiring.package=org.osgi.framework)(version>=1.6.0)(!(version>=2.0.0)))
```
Comment (jbonofre) [inline features/extension/src/test/resources/wirings/9]: This file doesn't contain the license header. You should exclude from rat configuration.

**Comment by nantunes** [comment]
Great. Let me know if some change is needed. ty

**Pull request closed without merging**